Formula/jsdoc3.rb

```diff
@@ -15,7 +15,7 @@ class Jsdoc3 < Formula
     sha256 "9d5a8b63309f02f86c42c7cf6973a000f4eda4ad0e1cea080a64674abea0a2da" => :el_capitan
   end
 
-  depends_on "node"
+  depends_on "node@10"
 
   def install
     system "npm", "install", *Language::Node.std_npm_install_args(libexec)
```

Comment:
```suggestion
     system "npm", "install", *Language::Node.std_npm_install_args(libexec)
     inreplace libexec/"lib/node_modules/jsdoc/node_modules/requizzle/lib/requizzle.js",
               "if (lookupPaths[0] === targetPath && lookupPaths[1].length === 0) {",
               "if (lookupPaths == null || lookupPaths[0] === targetPath && lookupPaths[1].length === 0) {"
```

and revert back the dependency to just `node`